src/main/java/net/moonbyte/AnarchyUtils/AnarchyUtils.java

```diff
@@ -56,18 +58,18 @@ public void onDisable() {
         log.info("Disabled Anarchy Utils!");
     }
 
-    public static AnarchyUtils getPlugin() {
+    public static @NotNull AnarchyUtils getPlugin() {
         return getPlugin(AnarchyUtils.class);
     }
 
     private void registerCommands() {
-        this.getCommand("nc").setExecutor(new ChangeNameColorCommand());
-        this.getCommand("discord").setExecutor(new DiscordCommand());
-        this.getCommand("help").setExecutor(new HelpCommand());
-        this.getCommand("kill").setExecutor(new KillCommand());
-        this.getCommand("togglejoinmsg").setExecutor(new ToggleJoinMessagesCommand());
-        this.getCommand("wiki").setExecutor(new WikiCommand());
-        this.getCommand("worldstatus").setExecutor(new WorldStatusCommand());
+        Objects.requireNonNull(this.getCommand("nc")).setExecutor(new ChangeNameColorCommand());
```

Comment:
I don't think we require Objects.requireNoonNull, look at other plugin source codes, they just call this.getCommand("commandName").setExecutor();